app/views/layouts/application.html.haml

```diff
@@ -57,3 +57,15 @@
 
     = include_chartbeat
     = include_mixpanel_guid
+
+    #blueimp-gallery.blueimp-gallery.blueimp-gallery-controls
```

Comment:
guess because the `#` is easier to notice than the `.`, so you're less likely to mistake `#foo.bar` for `#foobar` or so.